command/server/params.go

```diff
@@ -108,6 +112,10 @@ func (p *serverParams) isDNSAddressSet() bool {
 	return p.rawConfig.Network.DNSAddr != ""
 }
 
+func (p *serverParams) isLogFileLocationSet() bool {
+	return p.rawConfig.LogFilePath != ""
```

Comment:
When do you actually set this value? `p.rawConfig.LogFilePath`

As far as I can see, you only manipulate `serverParams.LogFilePath`

Reply:
Good catch. Fixed in fe41b1e3945aa3ff7c35ff6095786ce8d717692c